### DISCUSSION TITLE
Advanced Frame Rate Options

### DISCUSSION THREAD

**Discussion by ClockTheSenior** [Sam54123/mc-world-export#52] [Ideas]
A lot can be said here, but I am going to simplify it. The user should be able to choose the frame rate in which to export or make it so that the animation playback framerate matches the blender project framerate. But since there is also a Blender Addon, you can add that feature to that.

**Comment by Igrium** [comment]
Frame rate isn't really a thing in this addon. Entities are always stored at 20 fps (one frame per tick) and on import, it uses the frame rate of your blender project

**Reply by ClockTheSenior** [reply]
Oh, you should probably put this in some sort of documentation or in the readme.md
